<launch>

  <!-- start a turtlebot in gazebo -->
  <include file="$(find turtlebot_gazebo)/launch/turtlebot_world.launch" />

  <!-- start our map server -->
  <arg name="JEEVES_LAUNCH_DIR" value="/home/rick/src/roboticsclub-mcecsbot/src/ros/launch" />
  <include file="$(arg JEEVES_LAUNCH_DIR)/map_server.launch" />

  <!-- fake amcl -->
  <include file="fake_localization.launch" />

  <!-- move_base -->
  <include file="$(find turtlebot_navigation)/launch/includes/move_base.launch.xml"/>
  <!-- default turtlebot is poorly-tuned-->
  <param name="move_base/DWAPlannerROS/max_rot_vel" value="1.0"/>
  <param name="move_base/DWAPlannerROS/acc_lim_theta" value="10.0"/>

  <!-- waypoint manager, and dev ui -->
  <include file="$(find jeeves_2d_nav)/launch/waypoint_manager.launch" />
  <include file="$(find jeeves_2d_nav)/web_ui/launch/web_ui.launch" />

</launch>
